src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Manage/Account.Manage.DeletePersonalData.cshtml

```diff
@@ -2,7 +2,7 @@
 @model DeletePersonalDataModel
 @{
     ViewData["Title"] = "Delete Personal Data";
-    ViewData["ActivePage"] = ManageNavPages.DeletePersonalData;
+    ViewData["ActivePage"] = ManageNavPages.PersonalData;
```

Comment:
Could you help me understand why we went to the less specific value here?

Reply:
This is to maintain parity with https://github.com/aspnet/Identity/issues/1866 and https://github.com/aspnet/Identity/pull/1838

The work was originally done in scaffolding here #829 but didn't get merged. I had to rework those changes after adding the bootstrap version support, which is why they're here.